tests/cpp/test_mma.cpp

```diff
@@ -1313,303 +790,6 @@ TEST_P(HopperSS, SingleTileTransposed) {
-  matmul_utils::moveInnerBroadcastLeft(tv0);
-  matmul_utils::moveInnerBroadcastLeft(tv1);
-
-  // Just schedule tv0 and tv1 the same way as in SingleTile. Note that although
-  // the schedule are the same, the memory layout is different.
-  // For example, assume that the inner size is 16, and the swizzle size is 64.
-  // For the case of SingleTile, the input tensor size will be 16, so the inner
-  // dimension will be split as:
-  //   1, 64 = split(16, 64)
-  // For the case of FullSwizzle, the input tensor size will be 64, so the inner
-  // dimension will be split as:
-  //   1, 64 = split(64, 64)
-  tv0->applyMmaSwizzle(swizzle_a);
-  tv1->applyMmaSwizzle(swizzle_b);
-
-  naivelyParallelize(tv0);
-  naivelyParallelize(tv1);
-
-  // [M, N, K]
-  int64_t inline_pos = 0;
-  if (multiple_a && m_is_inner) {
-    tv2c->split(-3, getM(macro));
-    tv2->split(-2, getM(macro));
-    inline_pos++;
-  }
-  if (multiple_b && n_is_inner) {
-    tv2c->split(-2, getN(macro));
-    tv2c->reorder({{-3, -4}});
-    tv2->split(-1, getN(macro));
-    tv2->reorder({{-2, -3}});
-    inline_pos++;
-  }
-  if ((multiple_a && !m_is_inner) || (multiple_b && !n_is_inner)) {
-    tv2c->split(-1, getK(macro));
-    tv2c->reorder({{-2, -4}});
-  }
-  // [Mo, No, Ko, Mi, Ni, Ki]
-
-  {
-    auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
-        tv2c->getLoopDomain());
-    tv2c->setAllocationDomain(s.as<IterDomain*>(), true);
-    // Note: according to internal doc "Representing ldmatrix", we need both a
-    // read domain and a write domain to correctly represent MmaOp. Without this
-    // new mechanism, there is no correct loop domain, and the only choices are
-    // either we want to represent the smem read well, or represent the register
-    // write well. We choose to represent the smem read well here. Likely, this
-    // means we will not be able to have multiple tiles in register, but we can
-    // workaround this by always inlining the MmaOp most. We should fix this
-    // after we implemented the new read/write domain mechanism.
-    tv2c->axis(-1)->parallelize(ParallelType::Mma);
-    tv2c->axis(-2)->parallelize(ParallelType::Mma);
-    tv2c->axis(-3)->parallelize(ParallelType::Mma);
-  }
-  {
-    auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
-        tv2->getLoopDomain());
-    tv2->setLoopDomain(s.as<IterDomain*>());
-  }
-
-  tv2c->inlineAt(inline_pos);
-
-  auto inputs =
-      matmulAtInput3DHopperSS(m, n, k1, layout, data_type_to_aten(dtype));
-
-  FusionExecutor fe;
-  fe.compileFusion(
-      &fusion, {inputs.first, inputs.second}, LaunchParams(), matmul_cparams);
-  auto cg_outputs = fe.runFusion({inputs.first, inputs.second});
-  auto tref = atMatmul(
-      inputs.first.squeeze().to(at::kFloat),
-      inputs.second.squeeze().to(at::kFloat),
-      layout);
-  EXPECT_TRUE(at::allclose(cg_outputs[0], tref, 1e-5, 1e-5));
-}
-
-TEST_P(HopperSS, SingleTileWithTMALoad) {
```

Comment:
We already have a test that uses TMA for both load and store. Although that test is `RS`, because the memory layout of A and B are the same, testing B should pretty confidently mean that A will just work.